ui/cypress/integration/volume/volumetabs.spec.js

```diff
@@ -44,6 +44,12 @@ describe('Volume page overview tab', () => {
       );
     });
   });
+
+  it('displays the LVMLogicalVolume type and VG Name', () => {
```

Comment:
We're missing "negative testing", basically:
- not showing "VG Name" on sparse volumes
- not showing "Backend Disk" on LV volumes

Can be handled as debt.

Reply:
Added the two scenarios ;)